request_handler.py

```diff
@@ -1,6 +1,7 @@
 from http.server import BaseHTTPRequestHandler, HTTPServer
 import json
 from views import create_user, login_user, get_all_categories
+from views import get_all_comments, get_single_comment, create_comment, delete_comment, update_comment
```

Comment:
This is a double import from line 3, please condense this import.

Reply:
I don't understand what you mean... line three does not have any of the same imports, right?